integration/cli_test.go

```diff
@@ -162,7 +162,7 @@ func TestCLI(t *testing.T) {
 		{
 			"fetch; jwt; no audience",
```

Comment:
"no scope or audience" here too

Reply:
This should be no scope since the fetch command uses OAuth as default. We will need to keep this unchanged for backwards compatibility.